test/spec/faros_event_spec.sh

```diff
@@ -35,23 +35,23 @@ Describe 'faros_event.sh'
           FAROS_RUN="<cicd_source>://<cicd_organization>/<cicd_pipeline>/<build_uid>" \
           FAROS_RUN_STATUS="Success" \
           FAROS_RUN_STATUS_DETAILS="<run_status_details>" \
-          FAROS_RUN_START_TIME="1" \
-          FAROS_RUN_END_TIME="2" \
+          FAROS_RUN_START_TIME="1000" \
+          FAROS_RUN_END_TIME="2000" \
           FAROS_DEPLOY="<deploy_source>://<app_name>/QA/<deploy_uid>" \
           FAROS_DEPLOY_APP_PLATFORM="<deploy_app_platform>" \
           FAROS_DEPLOY_ENV_DETAILS="<deploy_env_details>" \
           FAROS_DEPLOY_STATUS="Success" \
           FAROS_DEPLOY_STATUS_DETAILS="<deploy_status_details>" \
-          FAROS_DEPLOY_START_TIME="3" \
-          FAROS_DEPLOY_END_TIME="4" \
+          FAROS_DEPLOY_START_TIME="3000" \
+          FAROS_DEPLOY_END_TIME="4000" \
```

Comment:
Should we test that it converts the other forms of timestamps at least once?

Reply:
Sounds good. I'll add a couple more tests